[MWAN] Routed networks / NAT (#13293)
cloudflare/cloudflare-docs: content/magic-wan/_partials/connector/_create-site.md

1. In **LAN configuration**, select **Create**.
2. Enter a descriptive name for your LAN in **Network name**.

> 3. **Physical port** $2

Reviewer: ```suggestion
3. **Physical interface** $2
```

(same change for WAN)

4. In **VLAN ID**, specify a [VLAN ID](/magic-wan/configuration/connector/reference/#vlan-id) to create virtual LANs.

> 5. In **Static addressing** give your Connector its IP address. You can also enable the following options if they suit your use case:

Reviewer 2: ```suggestion
5. In **Static address** give your Connector its IP address. You can also enable the following options if they suit your use case:
```
I _believe_ the field is **Static address**. Disregard if it's not.

Reviewer: There's a section called "Static Addressing," and the first field in that section is "Static Address". Not sure which we want to use here (or both).

> 5. In **Static addressing** give your Connector its IP address. You can also enable the following options if they suit your use case:

Reviewer: ```suggestion
5. In **Static addressing** give your Connector's LAN interface its IP address. You can also enable the following options if they suit your use case:
```

    - **This is a DHCP server**: If your Connector is a [DHCP server](/magic-wan/configuration/connector/network-options/dhcp/dhcp-server/).
    - **This is a DHCP relay**: If your Connector is a [DHCP relay](/magic-wan/configuration/connector/network-options/dhcp/dhcp-relay/).

> 6. (Optional) In **Static NAT prefix**, enter an IP address if you plan to enable NAT (network address translation). The IP address you enter here should be the same size of the prefix entered in **Static addressing**. For example, both networks have a subnet mask of `/24`: `192.168.100.0/24` and `10.10.100.0/24`.

Reviewer 2: ```suggestion
6. (Optional) In **Static NAT prefix**, enter an IP address if you plan to enable NAT (network address translation). The IP address you enter here should be the same size of the prefix entered in **Static address**. For example, both networks have a subnet mask of `/24`: `192.168.100.0/24` and `10.10.100.0/24`.
```

> 7. (Optional) If your LAN contains additional subnets behind a layer 3 router, select **Add routed subnet** under **Routed subnet** to add them:

Reviewer: ```suggestion
7. (Optional) If your LAN contains additional subnets behind a layer 3 router, select **Add routed subnet** under **Routed subnets** to add them:
```

    - **Prefix**: The CIDR prefix for the subnet behind the L3 router.
    - **Next hop**:  The address of the L3 router to which the Connector should forward packets for this subnet.

>     - **Static NAT prefix**: Optional setting. If you want to enable NAT for a routed subnet, you need to have an "external" prefix for the overlay-facing side of the NAT to use.

Reviewer: ```suggestion
    - **Static NAT prefix**: Optional setting. If you want to enable NAT for a routed subnet, supply an "external" prefix for the overlay-facing side of the NAT to use. It must be the same size as **Prefix**.
```
